bluemira/base/reactor.py

```diff
@@ -503,6 +503,14 @@ def _build_component_tree(
         with_components: Optional[List[ComponentManager]] = None,
     ) -> Component:
         """Build the component tree from this class's annotations."""
+        if not hasattr(self, "__annotations__"):
+            raise ComponentError(
+                "This reactor is ill-defined. "
+                "Have you sub-classed Reactor and "
+                "correctly defined the component managers for it? "
+                "Please look at the examples for how to do this."
```

Comment:
As you say I think we can reword this. It feels a little condescending as is

possibly:
```suggestion
                "This reactor is ill-defined. "
                "Make sure you have sub-classed Reactor and "
                "correctly defined component managers for it. "
                "Please see the examples for a template Reactor."
```